concepts/observability-strategy/configurable-tracing/03-tracing-pocs.md

```diff
@@ -32,7 +32,26 @@ Comparing the resource consumption and throughput of the envoys, and checking ot
 
 ## Tail Sampling
 
-Understand what tail sampling strategies are available already in OTEL and understand the scaling requirements of them.
+Opposed to Head-based sampling, Tail sampling make the decision at the end of the entire flow, wenn whole trace data already gathered. This kind of sampling decision made at the collector level.
+
+With Tail sampling, it's possible to create advanced rules to filter out traces based on any **span** property, include their attributes, duration etc. Tail sampling will allow us to collect data like unsually long operations and rare errors.
+
+Basically, with Tail sampling, sampling decision delayed end of flow untill all spans of a trace are available, this enables better sampling decisions based on all data from the trace.
```

Comment:
```suggestion
With the sampling decision delayed to the end of flow when all spans of a trace are available, the decision is better because it is based on all data from the trace.
```